Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -188,7 +188,10 @@ public class JavaScriptLifter: Lifter {
 
             case is CreateArray:
                 // When creating arrays, treat undefined elements as holes. This also relies on literals always being inlined.
-                let elems = instr.inputs.map({ let text = expr(for: $0).text; return text == "undefined" ? "" : text }).joined(separator: ",")
+                var elems = instr.inputs.map({ let text = expr(for: $0).text; return text == "undefined" ? "" : text }).joined(separator: ",")
+                if instr.inputs.last != nil && expressions[instr.inputs.last!] != nil && expressions[instr.inputs.last!]!.text=="undefined" {
```

Comment:
I think this can be simplified a bit, for example to something like
```swift
if elems.last == "," {
    // If the last element is supposed to be a hole, we need two commas
    elems += ","
}
```

Reply:
i changed it to the simplified version of code and added the test~